components/radio/stories/radio.stories.js

```diff
@@ -37,6 +37,7 @@ export default {
 			control: { type: "text" },
 		},
 		isEmphasized,
+		isInvalid,
```

Comment:
I'd also import and add `isFocused` here, along with a default arg value of `false`, since that's being used now.